modifiers.cwt

```diff
@@ -566,3 +566,11 @@ alias[modifier:enum[troop_types_land_only_defensive]] = float
 alias[modifier:<special_troop_type>_morale] = float
 # scope = unit
 alias[modifier:enum[troop_types_land_only_morale]] = float
+
+# TODO: Docs
+# scope = character
+alias[modifier:court_size_modifier] = int
```

Comment:
Add these into the modifiers block above. Only needs to be done through alias if complicated (e.g. uses type in name)

Reply:
Cool thanks